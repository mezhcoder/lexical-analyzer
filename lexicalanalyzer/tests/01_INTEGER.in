101 555
2003
404
101
5