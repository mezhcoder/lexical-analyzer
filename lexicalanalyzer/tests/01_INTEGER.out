1	1	INTEGER	101	101
1	5	INTEGER	555	555
2	1	INTEGER	2003	2003
3	1	INTEGER	404	404
4	1	INTEGER	101	101
5	1	INTEGER	5	5